docs/source/quickstart-pt-2.mdx

```diff
@@ -0,0 +1,172 @@
+
+Run the following from your project directory, **substituting your Studio graph's name where indicated**:
+
+```shell
+rover subgraph publish <GRAPH_NAME> \
+  --routing-url https://7bssbnldib.execute-api.us-east-1.amazonaws.com/Prod/graphql \
+  --schema ./products.graphql \
+  --name products
+```
+
+> Technically, you don't provide the graph's _name_, but rather its **graph ref**. [Learn more.](https://www.apollographql.com/docs/rover/essentials/#graph-refs)
+
+If the command is successful, you'll see output like the following:
+
+```
+A new subgraph called 'products' for the 'docs-example-graph' graph was created
+The gateway for the 'docs-example-graph' graph was updated with a new schema, composed from the updated 'products' subgraph
+```
+
+Nice! If you open your graph's details in Studio now, you'll see types and fields from our `products` subgraph listed in the Schema tab.
+
+Now, let's do the same thing for our `reviews` service, **again substituting your graph's name where indicated**:
+
+```shell
+rover subgraph publish <GRAPH_NAME> \
+  --routing-url https://w0jtezo2pa.execute-api.us-east-1.amazonaws.com/Prod/graphql \
+  --schema ./reviews.graphql \
+  --name reviews
+```
+
+If you refresh the Schema tab in Studio, you'll now see types and fields from our `reviews` service as well.
+
+Our supergraph schema has now been generated automatically by Apollo, but our _gateway_ doesn't know how to poll for it yet. Let's tackle that next.
+
+## 4. Authenticate the gateway with Apollo Studio
+
+Right now in `index.js`, we're providing our supergraph schema to our `ApolloGateway` via the `supergraphSdl` option. We don't need to do this anymore!
+
+Replace the contents of `index.js` with the following:
+
+```js:title=index.js
+const { ApolloServer } = require('apollo-server');
+const { ApolloGateway } = require('@apollo/gateway');
+
+const gateway = new ApolloGateway({});
+
+const server = new ApolloServer({
+  gateway,
+  // Subscriptions are not currently supported in Apollo Federation
+  subscriptions: false
+});
+
+server.listen().then(({ url }) => {
+  console.log(`🚀 Gateway ready at ${url}`);
+}).catch(err => {console.error(err)});
+```
+
+So now, how do we connect and authenticate our gateway with Apollo Studio? With a **graph API key** that we set as the value of an **environment variable**.
+
+> **API keys are secret credentials.** Never share them outside your organization or commit them to version control. Delete and replace API keys that you believe are compromised.
+
+1. Obtain a graph API key for your Studio graph by <a href="https://www.apollographql.com/docs/studio/api-keys/#graph-api-keys" target="_blank">following these steps</a>.  Copy its value.
+
+2. Create a new file named `.env` in your Node.js project folder.
+
+3. Paste the following into `.env` and replace `YOUR_API_KEY` with your graph API key:
```

Comment:
this step could possibly work for rover's authentication as well, since we also use `APOLLO_KEY`. You'd just need to make sure the `.env` is actually sourced prior to running `rover`, which would be a slightly different flow than the `require('dotenv').config()` that we have below. might be more trouble than its worth to test to get working but thought it was worth mentioning

Reply:
ah neat! Yeah I think we'll roll with what we have for simplicity's sake but thanks for raising!